tests/console/commands/env/conftest.py

```diff
@@ -36,7 +36,7 @@ def python_versions() -> list[str]:
 
 @pytest.fixture
 def venvs_in_cache_config(app: PoetryTestApplication, venv_cache: Path) -> None:
-    app.poetry.config.merge({"virtualenvs": {"path": str(venv_cache)}})
+    app.poetry.config.merge({"virtualenvs": {"path": venv_cache}})
```

Comment:
```suggestion
    app.poetry.config.merge({"virtualenvs": {"path": str(venv_cache)}})
```

I'd say let's keep `str` for consistency. By the way that will trigger CI. 😉 (One extra run is enough for me since I'm quite confident about the fix.)

Reply:
:heavy_check_mark: